docs/structure/sections-headers.md

```diff
@@ -0,0 +1,126 @@
+represents the structure of your book in a particular way. Different choices
+for the structure of `_toc.yml` and the header structures within your pages will
+result in different outcomes for your overall book structure. Here are some general
+tips and best-practices.
+
+```{note}
+This is particularly important when you [number your book's sections](toc/numbering)
+or when you [build a PDF of your book through Latex](pdf:latex).
+```
+
+**Chapters are at the top of your book hierarchy**. The top level of your `_toc.yml` contains
+a list of chapters. The title of each file will be the chapter's title.
+
+**Headers map onto sections**. Jupyter Book interprets your book as a collection of sections,
+and decides how those sections should be nested according to the hierarchy of
+`_toc.yml` and the hierarchy of headers in a page. Within a file, the first
+`## ` header it discovers will define the top-most section in the file, and any subsequent
+`### ` headers underneath will become sub-sections (until another `## ` section
+is encountered). This behavior is a bit different if the page is *nested* under
+another (see below).
+
+**Nested files define sections _underneath_ the last section of their parent**.
+If you specify sections that are *nested* under a file (with the `sections:` key)
+then those sections will begin *underneath* the last headers of the parent page.
+
+For example, if your `_toc.yml` file looks like this:
+
+```yaml
+format: jb-book
+root: myintro
+chapters:
+- file: chapter1
+  sections:
+  - file: chapter1section
+```
+
+Then the sections of `chapter1section` will begin **under** the sections of `chapter1`.
+Any headers in `chapter1section` will be treated as a "next-header-deeper" section in
+`chapter1`.
+
+In other words, if `chapter1` and `chapter1section` look like this:
+
+````{panels}
+`chapter1.md`
+^^^^^^^^^^^^^
+```md
+# Chapter 1 title
+
+## Chapter 1 second header
+```
+---
+`chapter1section.md`
+^^^^^^^^^^^^^^^^^^^^
+
+```md
+# Chapter 1 section title
+
+## Chapter 1 section second header
+```
+````
+Then your book will treat them like so:
+
+```md
+# Chapter 1 title
+
+## Chapter 1 second header
+
+### Chapter 1 section title
+
+#### Chapter 1 section second header
+```
+
+However, if `chapter1.md` had an extra third-level header, like so:
+
+````{panels}
+`chapter1.md`
+^^^^^^^^^^^^^
+```md
+# Chapter 1 title
+
+## Chapter 1 second header
+
+### Chapter 1 third header
+```
+---
+`chapter1section.md`
+^^^^^^^^^^^^^^^^^^^^
+
+```md
+# Chapter 1 section title
+
+## Chapter 1 section second header
+```
+````
+
+Then your book would treat them like so:
+
+```md
+# Chapter 1 title
+
+## Chapter 1 second header
+
+### Chapter 1 third header
+
+#### Chapter 1 section title
+
+##### Chapter 1 section second header
+```
+
+Keep this in mind when you design the structure of your files.
+
+:::{tip}
+A good rule of thumb is to take one of these two approaches:
+
+1. **don't put headers in your introduction pages**. This is
```

Comment:
Note that jupyterbook-latex essentially removes all headers in the introduction (aka root) document, maeaning that its child documents will all start at H2

Reply:
Interesting...do you think we should document this differently? Should it block this PR? (or if not, perhaps we should open an issue to clarify this point?)

Reply:
Here is the `pdflatex` artifact that was build on the latest `master` re: behaviour of `jupyterbook-latex` around the front page (root document). 

https://github.com/executablebooks/jupyter-book/suites/2660999633/artifacts/58566503

It essentially treats the `root` page as `front-matter` (loosely)